muzzle/src/test/java/io/opentelemetry/javaagent/tooling/muzzle/ReferenceCollectorTest.java

```diff
@@ -259,61 +259,6 @@ public void shouldCorrectlyFindExternalInstrumentationClasses() {
     assertThat(helperClasses).containsExactly(ExternalHelper.class.getName());
   }
 
-  @ParameterizedTest
-  @MethodSource
-  public void shouldCollectHelperClassesFromResourceFile(
```

Comment:
We should not remove that test; it should be rewritten to the new version instead.